config/v1/types_infrastructure.go

```diff
@@ -91,6 +91,9 @@ const (
 
 	// VSpherePlatformType represents VMWare vSphere infrastructure.
 	VSpherePlatformType PlatformType = "VSphere"
+
+	// OvirtPlatformType represent oVirt/RHV infrastructure.
+	OvirtPlatformType PlatformType = "Ovirt"
```

Comment:
`OVirtPlatformType PlatformType = "oVirt"`?

Reply:
> I hope that when we compare platform we ignore cases.

Most code I've uses `switch` or equality checks without `Lower` wrapping.  If we expect folks to use case-insensitive comparison (I'm fine either way), we'll probably want to land strong language about it here.  And also find a way to test non-canonical casing vs. our e2e suite.